docs/docs/20-usage/50-environment.md

```diff
@@ -80,7 +80,8 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_PIPELINE_NUMBER`             | pipeline number                                                                              |
 | `CI_PIPELINE_PARENT`             | number of parent pipeline                                                                    |
 | `CI_PIPELINE_EVENT`              | pipeline event (push, pull_request, tag, deployment)                                         |
-| `CI_PIPELINE_URL`                | link to the forge's web UI for the commit(s) or tag that triggered the pipeline              |
+| `CI_PIPELINE_URL`                | link to the web UI for the pipeline                                                          |
+| `CI_PIPELINE_FORGE_URL`          | link to the forge's web UI for the commit(s) or tag that triggered the pipeline              |
```

Comment:
Why do we need it if we already have `CI_COMMIT_URL`?

Reply:
As done with other env vars in the past, deprecated it and remove it in v3?